src/less/toast/base/toast.less

```diff
@@ -0,0 +1,97 @@
+:root {
+    --ebay-skin-toast-spacing: 24px;
+}
+
+.toast__mobile {
+    --ebay-skin-toast-spacing: 16px;
+}
+
+.transition() {
+    transition:
+        opacity 0.2s cubic-bezier(0.21, 0.31, 1, 1.22) 0s,
+        transform 0.2s cubic-bezier(0.21, 0.31, 1, 1.22) 0s;
+}
+
+.toast {
+    background-color: @toast-box-color;
+    bottom: 16px;
+    color: @white-color;
+    left: 16px;
+    max-height: 40vh;
+    max-width: 480px;
+    min-width: 320px;
+    position: fixed;
+    z-index: 1;
+
+    &.toast--transition {
+        display: block;
```

Comment:
Why not put the transitions here? That way you can only do it once and no need for mixin.